### PR TITLE
MAGECLOUD-3054: Implement Database Locks using Zookeeper and flock fo…

### PR DESCRIPTION
**Pull request by meker12**
## This PR is a:

- [x] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

These documentation updates are associated with the following Magento Core code PRs:
- [2.3.2](https://github.com/magento/magento2ce/pull/3933)
- [2.2.9](https://github.com/magento/magento2ce/pull/3943)

## Additional information

*List all affected URLs*

- https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/config-reference-envphp.html
- https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/install-cli-install.html
- https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/install-cli-subcommads-deployment.html
- https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/Install-cli-subcommands-lock.html

whatsnew
Added a feature to configure the lock provider to prevent the launch of duplicate cron jobs and cron groups. In previous releases, Magento used the database to save locks. Now you have the option to manage locks using the database, Zookeeper, or file locks. See [Configure the lock provider](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-cloud.html).

### CODE REVIEW

**Comment by meker12** [comment]
These changes originally submitted in https://github.com/magento/devdocs/pull/4137 against the Cloud-ece-2.2.9 branch.  Picked and squashed changes to this PR to isolate changes to submit directly to `develop` branch for 2.2.9 release.

File: _data/toc/installation-guide.yml [hunk 1]
```diff
@@ -258,6 +258,10 @@ pages:
             - label: Update the Magento database schema and data
               url: /install-gde/install/cli/install-cli-subcommands-db-upgr.html
 
+            - label: Configure the lock provider
+              url: /install-gde/install/cli/install-cli-subcommands-lock.html
+              include_versions: ["2.1","2.2","2.3"]
```
Comment (jeff-matthews) [inline _data/toc/installation-guide.yml]: I don't see a 2.1 file. Should this be limited to 2.2 and 2.3?

Comment (meker12) [inline _data/toc/installation-guide.yml]:
```suggestion
              include_versions: ["2.2","2.3"]
```

Comment (meker12) [reply _data/toc/installation-guide.yml]: Only supported on 2.2.5 and later. Thx for catching that.

**Comment by QuentinFarizonAfrimarket** [comment]
Hello,
Can you point to commits in https://github.com/magento/magento2/ implementing this feature ?

**Pull request merged**